### PR TITLE
Add lich phylactery to list of POIs

### PR DESCRIPTION
**Pull request by dearmochi**
## What Does This PR Do

- Adds lich phylactery to list of POIs

## Why It's Good For The Game

To find out where the hell it is as observer

## Testing

trust

## Changelog
:cl:
add: Lich phylactery can now be found in the observe menu
/:cl:

### CODE REVIEW

File: code/datums/spells/lichdom.dm [hunk 1]
```diff
@@ -142,3 +145,7 @@
 	H.equip_to_slot_or_del(new /obj/item/clothing/head/wizard/black(H), SLOT_HUD_HEAD)
 	H.equip_to_slot_or_del(new /obj/item/clothing/shoes/sandal(H), SLOT_HUD_SHOES)
 	H.equip_to_slot_or_del(new /obj/item/clothing/under/color/black(H), SLOT_HUD_JUMPSUIT)
+
+/datum/spell/lichdom/proc/on_marked_item_destroy(obj/item/target)
+	SIGNAL_HANDLER
```
Comment (Contrabang) [inline code/datums/spells/lichdom.dm:150]:
```suggestion
	SIGNAL_HANDLER // COMSIG_PARENT_QDELETING
```

**Comment by Gatchapod** [comment]
Won't that let anyone with a scrying orb from the Give Magic ritual instantly locate it?
Not like wizards are forced to use that ritual along with the Lich spell, but I feel like it may be soft locking the option out.

**Comment by Qwertytoforty** [comment]
I really do not think that ghosts need an option to directly orbit the lich item, especially since people have meta'd the item in the past. Yes, it's against the rules, but it can be a bitch to confirm at times. I would prefer if it was not showing on orbit, especially when ghosts are muted or such. If an admin needs to find the item, it could be in check antags, or we could VV the spell

**Comment by dearmochi** [comment]
I forgot scrying orb was a thing 😔

**Pull request closed without merging**